lib/platform/index.ts

```diff
@@ -35,6 +35,9 @@ export function setPlatformApi(name: string): void {
     );
   }
   _platform = platforms.get(name);
+  if (!_platform.commitFiles) {
```

Comment:
This looks like a hack. Can't we instead add an optional refresh PR function to platform API?

So we can call it after commitFiles. Like `platform?.refreshPr(number)`.

Reply:
Why `platform?.` ? Shouldn't it always be defined - it's `refreshPr` that would be optional?

Reply:
Sorry must be `platform.refreshPr?.(...)` 🙃